### ISSUE TITLE
config.json wiped out

### ISSUE THREAD

**Issue by mochetts** [mattermost/mattermost#27299]
Per Mattermost guidelines, GitHub issues are for bug reports: <https://handbook.mattermost.com/contributors/contributors/ways-to-contribute/>.

For troubleshooting see: https://forum.mattermost.com/.
For feature proposals see: https://mattermost.com/suggestions/

If you've found a bug--something appears unintentional--please follow these steps:

1. Confirm you’re filing a new issue. [Search existing tickets in Jira](https://mattermost.atlassian.net/jira/software/c/projects/MM/issues/) to ensure that the ticket does not already exist.
2. Confirm your issue does not involve security. Otherwise, please see our [Responsible Disclosure Policy](https://mattermost.com/security-vulnerability-report/).
3. [File a new issue](https://github.com/mattermost/mattermost/issues/new) using the format below. Mattermost will confirm steps to reproduce and file in Jira, or ask for more details if there is trouble reproducing it. If there's already an existing bug in Jira, it will be linked back to the GitHub issue so you can track when it gets fixed. 

#### Summary
config.json file gets wiped out

#### Steps to reproduce
- Install mattermost using bitnami in gcp
- config.json file gets wiped out after a while 

#### Expected behavior
The config file shouldn't be wiped out. This is causing the mattermost server to fail.

#### Observed behavior (that appears unintentional) 
![Screenshot 2024-06-07 at 1 48 23 PM](https://github.com/mattermost/mattermost/assets/3678598/198749ac-8fd8-4fbf-b1f0-1eac09766063)

#### Possible fixes
n/a

**Comment by amyblais** [comment]
Opened https://mattermost.atlassian.net/browse/MM-59481.

**Comment by agnivade** [comment]
@mochetts - How are you installing Mattermost? Are you using our official tarball builds or some other method?

And to clarify - are you seeing the config.json being wiped out while Mattermost is running? Or does this happen when you restart the server?

**Comment by mochetts** [comment]
I was using the bitnami google cloud bundle. It happened while Mattermost was running. If I stopped the server it wouldn't happen.

**Comment by agnivade** [comment]
I am not aware of the "bitnami google cloud bundle". Could you point me to that? Just want to make sure that there isn't anything else that's messing up with the installation.

I am wondering if you would be able to set an `strace` for the mattermost process and monitor for file system calls to the config.json file? That would let us identify what is really going on. I assume you haven't able to figure out any deterministic steps that causes this?

**Comment by mochetts** [comment]
This is the bitnami package I'm referring to:
https://bitnami.com/stack/mattermost

Recommended by the mattermost docs:
https://docs.mattermost.com/install/deploying-team-edition-on-bitnami.html

**Comment by mochetts** [comment]
Regarding the `strace` request, I'm not able to since I destroyed that VM and reinstalled mattermost on a fresh one. The issue was kind of critical as it put the company to a communication halt.  Since I installed it on a fresh VM instance, the issue didn't occur.
One thing to point out tho, In the original VM I installed the [Remind Plugin](https://mattermost.com/marketplace/remind-plugin/). I didn't install it in the fresh new one as I thought that plugin might have been the cause.
